akka-docs/src/main/paradox/stream/operators/Source-or-Flow/sliding.md

```diff
@@ -4,20 +4,47 @@ Provide a sliding window over the incoming stream and pass the windows as groups
 
 @ref[Simple operators](../index.md#simple-operators)
 
-@@@div { .group-scala }
-
 ## Signature
 
-@@signature [Flow.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala) { #sliding }
-
-@@@
+@apidoc[Flow.sliding](Flow) { scala="#sliding(n:Int,step:Int):FlowOps.this.Repr[scala.collection.immutable.Seq[Out]]" java="#sliding(int,int)" } 
```

Comment:
How do you work these out easily? Is it the same as the link in the API doc?

Reply:
I have same question. This approach seems error prone and difficult for contribitors and us to figure out. When I originally proposed linking to apidoc the idea was that users would have to go to the apidocs to see the signature. Link would be something like Flow#sliding

Reply:
So it is quite straightforward as you go to the api doc and copy the #xxx bit from the URL.

We can add something to the issue/contributing docs for how to do this

It still only produces a link to the API doc.

Reply:
Link looks like that `Flow.sliding` the tricky part is just there to make it lead to the right section in the Scaladoc vs the Javadocs. It's the anchor part so I "just" go to to the respective API:docs and copy that.

Reply:
I agree it is suboptimal, if someone has a better solution I'm all ears. 👂👂👂👂